src/FHIRSearchParametersRegistry/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { FhirVersion } from 'fhir-works-on-aws-interface';
+import compiledSearchParamsV4 from '../schema/compiledSearchParameters.4.0.1.json';
+import compiledSearchParamsV3 from '../schema/compiledSearchParameters.3.0.1.json';
+
+export type SearchParam = {
```

Comment:
Could you add comments explaining the params;; if you use long winded comments, like :
```suggestion
/**
* <comment here>
*/
export type SearchParam = {
```

My IDE (VSCode) will add those comments when I hover over time object

Reply:
added docs